### PR TITLE
gtkdialog: switch to new release

### PR DESCRIPTION
**Pull request by privatezero**
This is happy on my end, but needs testing on the mac side of things!

### CODE REVIEW

**Review by retokromer — COMMENTED** [review]
- Tested successfully on Debian 10.8.
- On macOS 10.14.6 I got:
```
==> Patching
Error: Failure while executing; `patch -g 0 -f -p1` exited with 1. Here's the output:
patching file src/automaton.c
Hunk #1 FAILED at 1295.
1 out of 1 hunk FAILED -- saving rejects to file src/automaton.c.rej
patching file src/variables.c
```

File: gtkdialog.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 class Gtkdialog < Formula
   desc "Small utility for fast and easy GUI building"
   homepage "https://code.google.com/archive/p/gtkdialog/"
-  url "https://storage.googleapis.com/google-code-archive-downloads/v2/code.google.com/gtkdialog/gtkdialog-0.8.3.tar.gz"
-  sha256 "ff89d2d7f1e6488e5df5f895716ac1d4198c2467a2a5dc1f51ab408a2faec38e"
-  revision 1
+  url "https://github.com/puppylinux-woof-CE/gtkdialog/archive/0.8.4c.tar.gz"
+  sha256 "fba198ca6aeb347de4e403751245a4c29faa1a57102a2b3413bda1690266ba06"
+  revision 2
```
Comment (retokromer) [inline gtkdialog.rb:6]: If you switch to the new version, then you should remove thee `revision` line.

File: gtkdialog.rb [hunk 2]
```diff
@@ -36,7 +36,7 @@ class Gtkdialog < Formula
   patch :DATA if OS.mac?
 
   def install
-    system "./configure"
+    system "./autogen.sh"
```
Comment (retokromer) [inline gtkdialog.rb:39]: I presume,
```rb
  depends_on "autoconf" => :build
  depends_on "automake" => :build
```
should be added?

**Comment by privatezero** [comment]
@dericed - any progress on the patch for mac? Or should I go ahead and make a PR with different sources for each platform?

**Comment by retokromer** [comment]
@privatezero @dericed I can test more on macOS, but if Dave can also test, that would be great! In fact, at the moment I don’t have access to “vanilla” macOS, but “only” to computers which are in production and have other pieces of software installed. In my personal opinion, different sources should be the plan B, only in case plan A in not possible.

**Review by retokromer — APPROVED** [review]
As a transitional solution until the new version is finalised and available also on macOS, I guess this is fine. Thank you!

If there is no objection, I will squash and merge it tomorrow.

**Review by dericed — APPROVED** [review]

**Comment by privatezero** [comment]
thanks @retokromer! Yes, if it works for everyone I'd very much support this getting squished and merged as an interim solution

**Pull request merged**